CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
     - `-experimental.tsdb.flush-blocks-on-shutdown` changed to `-experimental.blocks-storage.tsdb.flush-blocks-on-shutdown`
 * [CHANGE] Flags `-bigtable.grpc-use-gzip-compression`, `-ingester.client.grpc-use-gzip-compression`, `-querier.frontend-client.grpc-use-gzip-compression` are now deprecated. #2940
 * [CHANGE] Limit errors reported by ingester during query-time now return HTTP status code 422. #2941
+* [FEATURE] Add support for azure storage of AzureChinaCloud,AzureGermanCloud,AzureUSGovernment. #2988
```

Comment:
Please move the CHANGELOG entry to the top, under "master / unrelease". This is the section for 1.3.0 which has already been cut. Thanks!

Reply:
done